package/piksi_system_daemon/src/main.c

```diff
@@ -61,6 +61,8 @@ static bool log_ping_activity = false;
 static const char const *ip_mode_enum_names[] = {"Static", "DHCP", NULL};
 enum { IP_CFG_STATIC, IP_CFG_DHCP };
 static u8 eth_ip_mode = IP_CFG_STATIC;
+static bool eth_locked;
```

Comment:
as discussed this will become an enum type `mode` with either `active` or `config` as options. `active` subs for `true` and `config` subs for `false`

Reply:
I think `interface mode` makes sense too help disambiguate it from the `ip config mode` we already have.